### DISCUSSION TITLE
Possibility to disable Cortex/Loki

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#37793] [Alerting]
I would like to have a way to disable Loki/cortex alert types or a way to disable those errors related to Cortex/Loki when the source doesn't support it

Reason: 
When not using cortex datasource we always receive some error message

**Comment by techwavedev** [comment]
already on 8.1.1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
